tests/queries/0_stateless/01233_check_part_meta_cache_in_atomic.sql

```diff
@@ -0,0 +1,126 @@
+-- Tags: no-parallel, no-fasttest
+
+-- Create table under database with engine atomic.
```

Comment:
Test looks more or less the same as 01233_check_part_meta_cache, maybe it worth rewrite it using jinja2 templates or even .sh test?

And by the way, by using .sh for test you can write it w/o predefined database and remove no-parallel tag.

Reply:
Ok, I will try.

Reply:
done. see tests/queries/0_stateless/01233_check_table_with_metadata_cache.sh